lib/SIL/SILPrinter.cpp

```diff
@@ -239,7 +239,8 @@ static void printValueDecl(ValueDecl *Decl, raw_ostream &OS) {
 
   if (Decl->isOperator()) {
     OS << '"' << Decl->getBaseName() << '"';
-  } else if (Decl->getBaseName() == "subscript") {
+  } else if (Decl->getBaseName() == "subscript" ||
+             Decl->getBaseName() == "deinit") {
```

Comment:
I'm not quite sure if we should escape `deinit` in SIL if used as a normal identifier. For subscripts we needed to, since otherwise the names of subscript getters/setters could have clashed with the getters/setters of instance variables named "subscript". AFAIK destructors in SIL always have the suffix `!destructor` so there's no name clash. Should we strive for consistency between the two keywords or simplicity on `deinit`?